STM32/LightController/Core/Src/LightController.c

```diff
@@ -63,8 +57,24 @@ typedef enum {
     WHITE
 }test_mode_state;
 
-static bool testState = false;
-static test_mode_state partyState = OFF;
+static bool isInTest = false;
```

Comment:
Ideally I would like to have a separate test system and not have a test sequence built into the board directly...

However, since it is therefore, I think your suggestion is the better option for now.

Reply:
Actually, on second thought I think we shouldn't make it a status bit. I think the next time we want to test a lightcontroller board we should make a test script sending standard commands to the board and see that it behaves as expected. 

As I recall we did this to save some time, but like all the other boards our test scripts should not exist in the board itself.

Reply:
Was it not Martin that ended up doing this?